src/include/planner/operator/logical_empty_result.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "logical_operator.h"
+
+namespace kuzu {
+namespace planner {
+
+class LogicalEmptyResult final : public LogicalOperator {
```

Comment:
Can u add a comment here this logical operator is currently only generated when we are doing the filter pushdown optimization? You can point to `FilterPushDownOptimizer::visitFilterReplace`, which I think can help understand this operator if someone runs into here without understanding the optimization.